### PR TITLE
Document dependency on Connext version for dynamic libs

### PR DESCRIPTION
**Pull request by samuelraeburn**
Requested by Thejas.
Screenshot from 2021-04-30 08-38-43

### CODE REVIEW

**Comment by samuelraeburn** [comment]
Screenshot from 2021-05-03 09-41-16

File: docs/release_notes.rst [hunk 1]
```diff
@@ -38,6 +38,8 @@ repository <https://github.com/rticommunity/rticonnextdds-connector>`__.
 Version 1.1.0
 ~~~~~~~~~~~~~
 
+*RTI Connector* 1.1.0 is built on `RTI Connext DDS 6.1.0 <https://community.rti.com/documentation/rti-connext-dds-610>`__.
```
Comment (samuelraeburn) [inline docs/release_notes.rst:41]: This sentence was missing here (was in JS). 
I do not know if it the correct link or not. @rkorte can you confirm?

File: docs/features.rst [hunk 2]
```diff
@@ -252,6 +252,7 @@ configure your environment so that Connector can load these additional libraries
   <Connext DDS installation directory>\lib\<architecture>\
 
 .. note::
-    Each version of Connector is compatible with a corresponding release of
-    Connext DDS Pro. It is not possible to load libraries from incompatible
-    Connext DDS Pro releases. Connector version 1.1.0 is only compatible with Connext DDS 6.1.0.
+    Each version of Connector can only load add-on libraries from its
+    corresponding Connext DDS release. You can see this correspondence in the
+    :ref:`release notes<Version 1.1.0>`. For example, Connector 1.1.0 can only
```
Comment (alexcamposruiz) [inline docs/features.rst:257]: I wanted to hard-code as few references to a specific version as possible. Can you link just to :ref:`release notes?

**Review by alexcamposruiz — CHANGES_REQUESTED** [review]

**Review by alexcamposruiz — APPROVED** [review]

File: docs/release_notes.rst (hunk 1 above)
Comment (rkorte) [reply docs/release_notes.rst:41]: Yes, that is a valid URL (for 6.1.0 documentation, anyway).

**Pull request merged**